packages/rule-typescript-config-is-valid/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@alrra can we agree on having all the typescript rules in one package (and thus one folder) or not?

Reply:
Go for it.

Reply:
should I change the name of the folder to `rule-typescript-config`?

Reply:
yes